tests/test_contributors_views.py

```diff
@@ -131,3 +134,17 @@ def test_get_contributors_from_parent(self):
             len(res.json['contributors']),
             2,
         )
+
+    def test_throttle_period_expired_no_timestamp(self):
+        is_expired = throttle_period_expired(timestamp=None,  throttle=30)
+        assert_true(is_expired)
+
+    def test_throttle_period_expired_using_datetime(self):
+        timestamp = datetime.utcnow()
+        is_expired = throttle_period_expired(timestamp=timestamp,  throttle=30)
+        assert_false(is_expired)
```

Comment:
Could you add a couple more assertions on these last two tests. Maybe timestamp + 29 seconds is false, and timestamp + 31 seconds is true? Just to ensure that the base functionality of the function works.